packages/css-engine/src/core/style-sheet-regular.test.ts

```diff
@@ -932,6 +932,35 @@ describe("Style Sheet Regular", () => {
     color: blue
   }
 }"
+`);
+  });
+
+  test("support descendent suffix", () => {
+    const sheet = createRegularStyleSheet();
+    sheet.addMediaRule("base", {});
+    const rule1 = sheet.addNestingRule(".instance");
```

Comment:
.instance or .element? technically you are talking about an element that is backing the instance, but in this case you are talking specifically about the element, instance is basically the component instance that contains that element, but you are not styling the instance

Reply:
Just a reference `[data-ws-id=instanceId]` -> `.instance`